nservicebus/azure-service-bus/error-handling.md

```diff
@@ -0,0 +1,50 @@
+---
+title: Azure Service Bus Error Handling
+reviewed: 2016-08-29
+tags:
+- Azure
+- Cloud
+- Error Handling
+---
+
+## Native Error Handling
```

Comment:
this is a good question, in the menu it for sure isn't, but in the search results it's usefull to distinguish the error handling related documents that exist (there is one for the core and maybe for other transports as well). I'd like to remove the 'Azure Service Bus' part as well because it eats up a lot of the allowed 60 characters, but didn't want to reduce the search experience